test/api/fixtures/reports.js

```diff
@@ -0,0 +1,137 @@
+var nock = require('nock');
+
+nock('https://devapi.currencycloud.com:443')
+    .post('/v2/authenticate/api', {
+        "login_id": "development@currencycloud.com",
+        "api_key": "deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef"
+    })
+    .reply(200, {"auth_token": "efa3f4d86206ef45263d7ce1dafeadbb"});
+
+//Create conversion report
+nock('https://devapi.currencycloud.com:443')
+    .post('/v2/reports/conversions/create', {
+        "buy_currency": "EUR",
+        "sell_currency": "GBP"
+    })
+    .reply(200, {
+        "id": "c2d7dd2c-9317-4517-9f5d-8d287a1c4ee7",
+        "short_reference": "RP-4270806-VLOIID",
+        "search_params": {
+            "buy_currency": "EUR",
+            "sell_currency": "GBP",
+            "scope": "own"
+        },
+        "report_type": "conversion",
+        "status": "processing",
+        "report_url": "",
+        "account_id": "c0ea266f-b969-4d61-9ca0-e790c79072e5",
+        "contact_id": "3cae081f-56f7-4f2d-8235-45713b592b43",
+        "created_at": "2018-11-05T15:12:45+00:00",
+        "updated_at": "2018-11-05T15:12:45+00:00"
+    });
+
+// Create payment report
+nock('https://devapi.currencycloud.com:443')
+    .post('/v2/reports/payments/create', {
+        description: 'TESTING REPORT FOR PAYMENTS',
+        currency: 'GBP',
+        amount_from: '1000',
+        amount_to: '10000'
+    })
+    .reply(200, {
+        "id": "41fc3cd8-5d7f-4c20-8b0c-58eb6d653695",
+        "short_reference": "RP-6523110-DNTMME",
+        "search_params": {
+            "currency": "GBP",
+            "scope": "own"
+        },
+        "report_type": "payment",
+        "status": "processing",
+        "report_url": "",
+        "account_id": "c0ea266f-b969-4d61-9ca0-e790c79072e5",
+        "contact_id": "3cae081f-56f7-4f2d-8235-45713b592b43",
+        "created_at": "2018-10-22T08:27:03+00:00",
+        "updated_at": "2018-10-22T08:27:03+00:00"
+    });
+
+// Find report requests
+nock('https://devapi.currencycloud.com:443')
+    .get('/v2/reports/report_requests/find')
+    .query({"per_page": "1"})
+    .reply(200, {
+        "report_requests": [{
+                "id": "195557f7-7f2a-47ff-92d7-b8662dd100fd",
+                "short_reference": "RP-7918625-XWJQIG",
+                "search_params": {
+                    "buy_currency": "EUR",
+                    "sell_currency": "GBP",
+                    "unique_request_id": "q1w2e3r4",
```

Comment:
unique_request_id should be in UUIDv4 format. For example: e1821376-6cb0-4204-bec7-552733f3ea5a